libs/cardano-ledger-test/src/Test/Cardano/Ledger/Generic/TxGen.hs

```diff
@@ -824,7 +824,7 @@ getConwayTxCertCredential (ConwayTxCertDeleg (ConwayRegCert _ _)) = Nothing
 getConwayTxCertCredential (ConwayTxCertDeleg (ConwayUnRegCert cred _)) = Just cred
 getConwayTxCertCredential (ConwayTxCertDeleg (ConwayDelegCert cred _)) = Just cred
 getConwayTxCertCredential (ConwayTxCertDeleg (ConwayRegDelegCert cred _ _)) = Just cred
-getConwayTxCertCredential (ConwayTxCertConstitutional _) = Nothing
+getConwayTxCertCredential (ConwayTxCertCommittee _) = Nothing -- TODO Get the credential
```

Comment:
Would it be better to have `error "Unimplemented"` here instead so we don't forget about it?

Reply:
Yeah, `error` would be better, but I want to implement it in this PR. I just need to ask @WhatisRT some questions about this first.

Reply:
As I suspected this case isn't relevant for `ConwayTxCertCommittee`, removed.